src/infinity-menu.js

```diff
@@ -266,7 +278,7 @@ export default class InfinityMenu extends React.Component {
 	 *  @description React render method for creating infinity menu
 	 */
 	render() {
-		const tree = dcopy(this.props.tree);
+		const tree = this.props.tree;
```

Comment:
The `isSearchDisplay` and `isSearchOpen` just we added to the object, even the user modifiles the tree, we should still able to display them correctly